content/documentation/pages/5-feature-guides/2-batch/3-partitioning.md

```diff
@@ -1,7 +1,260 @@
 ---
 path: 'feature-guides/batch/partitioning/'
-title: 'Data Partitioning'
-description: 'How to use partitioning'
+title: 'Remote Batch Partitioning'
+description: 'How to use remote batch partitioning'
 ---
 
-Introduction to partitioning
+# Introduction to remote partitioning
+
+The Batch Developer Guide showed you how to create the typical single threaded Spring Batch application.
+While this is a great solution for most batch applications, there are cases where a particular step may take a significant amount time to do the work required.
+Spring Batch has a solution that allows a [Batch Job](https://docs.spring.io/spring-batch/4.0.x/reference/html/domain.html#job) to partition a [Step](https://docs.spring.io/spring-batch/4.0.x/reference/html/domain.html#step) execution, where each partition handles a segment of work.
+In short partitioning allows multiple versions of large batch applications to run concurrently.
+The purpose of this is to reduce the elapsed time required to process long batch jobs.
+Processes that can be successfully partitioned are those where the input file can be split and/or the main database tables partitioned to allow the application to run against different sets of data.
+
+So lets say we have a 3 Step Job:
+
+![Batch Partitioning](images/batch-partition.png)
```

Comment:
I see that we are showcasing the example from [samples](https://github.com/spring-cloud/spring-cloud-task/tree/master/spring-cloud-task-samples/partitioned-batch-job); however, the visual is showing something about the usage aggregation. Should we instead put a visual to highlight what is happening in the sample?

Reply:
Agreed.  Image updated.

Reply:
Thanks for looking into this! So, I see:

![image](https://user-images.githubusercontent.com/1562654/57889839-bee14e00-77ea-11e9-99ce-c424fb9321c2.png)

I'm not sure the labels in the worker boxes are relevant to the example that we are showing here. It could be that I'm reading things differently. Sorry, if that is so.

Reply:
DOH!  Forgot to remove the `Aggregate Usage Information` stuff

Reply:
Thanks for catching that.  Image is updated